pyrasterframes/src/main/python/tests/GeoTiffWriterTests.py

```diff
@@ -29,28 +29,34 @@ class GeoTiffWriter(TestEnvironment):
 
     @staticmethod
     def _tmpfile():
-        return os.path.join(tempfile.tempdir, "pyrf-test.tif")
+        return os.path.join(tempfile.gettempdir(), "pyrf-test.tif")
```

Comment:
Is this a more stable way in Python?

Reply:
I am not 100% sure but i was having problems with `tempdir`